web/modules/custom/joinup_federation/src/Plugin/Adms2ConvertPass/Pass41.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\joinup_federation\Plugin\Adms2ConvertPass;
+
+use Drupal\KernelTests\KernelTestBase;
+use Drupal\joinup_federation\JoinupFederationAdms2ConvertPassPluginBase;
+use Drupal\Tests\joinup_federation\Kernel\ConvertToAdms2StepTest;
+
+/**
+ * Conversion Pass #41.
+ *
+ * URI: dct:issued
+ * Type: N.A.
+ * Action: Updated
+ * Description:
+ * - The cardinality changed to single valued. Only the first value will be
+ * kept.
+ *
+ * @Adms2ConvertPass(
+ *   id = "pass_41",
```

Comment:
We decided to take the "conservative" approach. Meaning that we can make any assumptions is such cases. If the incoming value have the cardinality of 1, then will pass the later ADMS-AP validation. But if the cardinality is >=2 then we cannot assume that we pickup the first, last, etc. Conclusion:
* Either remove this plugin/file.
* Or remove the annotation so that the plugin manager will not pick it up, add a `@todo` explaining the reason.